R-package/NEWS.md

```diff
@@ -0,0 +1,32 @@
+
+
+## RGF 1.0.4
+
+* I modified the license from GPL-3 to MIT to go in accordance with the new structure of the *rgf_python* package. The package includes two files : *LICENSE* (for the RGF R package) and *LICENSE.note* (for the *RGF*, *FastRGF* and *rgf_python* packages).
```

Comment:
Please rewrite this file either in passive voice, or at least with "we" instead of "I"

Reply:
I added "We (RGF-team) ...." for this first case and I pushed the changes.

Reply:
You are probably right. I converted all 'I' to 'We'

Reply:
Thank you!